microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -33,42 +95,86 @@ To submit suggestions and feedback, enter an [Issue about the Add-ons API](https
+
+##### [v1](#tab/v1)
+
+1. At your [Partner Center developer dashboard](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd), sign in to the account that you used to publish an add-on.
+
+1. Under the **Microsoft Edge** program, select **Publish API**.
+
+1. On the **Publish API** page, click the **Create API credentials** button.  This step may take a few minutes to finish.
+
+   The API credentials have now been created; you've enabled or renewed the API.  The following items are now displayed on the **Publish API** page:
+   * Your **Client ID**.
+   * Your **Access token URL**.
+   * **Client secrets**.
+      * The **Expiry date** of each Secret.
 
    ![The 'Publish API' page at Partner Center after clicking 'Create API credentials', now showing Client ID, Client Secret, and Auth Token URL](./using-addons-api-images/create-api-credentials-button.png)
 
-1. Write down the **Client ID**, **Client secret** and the **Access token URL**.  You'll use these values in the next step, to get an access token.
+1. Write down the following:
+   * The **Client ID**.
+   * The **Access token URL**.
+   * The new **Client secret**.
+
+   You'll use the Client ID, access token URL, and client secret to get an access token, per [Retrieve the access token](#retrieve-the-access-token), below .  You'll then use the access token in the request header when using a REST endpoint.
 
 > [!IMPORTANT]
-> Be sure to write down the client secret now, because it's only visible immediately after enabling or renewing the API (that is, after creating API credentials). This particular secret isn't shown again.
+> Be sure to write down the client secret now, because it's only visible immediately after enabling or renewing the API (that is, after creating API credentials).  This particular secret isn't shown again.
 
-You can generate multiple client secrets for your client ID.  For example, you can create multiple secrets for multiple projects.
+You can generate multiple client secrets for your Client ID.  For example, you can create multiple secrets for multiple projects.
 
 
-<!-- ====================================================================== -->
-## Retrieving the access token
+<!-- ---------------------------------- -->
+#### Retrieve the access token
+<!-- todo: global: access token, or access token URL?  consistency pass, relate the two terms -->
+
+After you've acquired the necessary authorization for your application, get access tokens<!-- todo: plural or singular?  one token per endpoint? --> for APIs.  To get a token by using the client credentials grant, send a POST request to the Access token URL (the OAuth token).
+
+The tenant information<!-- todo: define "tenant information", what relevance? --> is available in the Access token URL that you received in the previous step, [Enable the Update REST API at Partner Center](#enable-the-update-rest-api-at-partner-center).
```

Comment:
define "tenant".  this is the only mention.  can sentence be greatly shortened? what info is included in tenant info?

Reply:
closing Outdated comment.  will enter fresh PR Comment at hidden todo comment.